backend/urls.py

```diff
@@ -20,6 +20,10 @@
          api_views.experiment_enrolls, name='v2-experiment-enrolls'),
     path('v2/experiments/<slug:exp_slug>/unenrolls/',
          api_views.experiment_unenrolls, name='v2-experiment-unenrolls'),
+    path('v2/experiments/<slug:exp_slug>/enrolls-full/',
+         api_views.experiment_enrolls_full, name='v2-experiment-enrolls-full'),
+    path('v2/experiments/<slug:exp_slug>/unenrolls-full/',
+         api_views.experiment_unenrolls_full, name='v2-experiment-unenrolls-full'),
```

Comment:
Can we name these to match the population ones (after rebase), renaming the old ones with "realtime" in the name?